### PR TITLE
Bold 4.8.3 Playstore version (#3) to master

### PR DESCRIPTION
**Pull request by KalmanPerkelGenesys**
* Bold latest version

* develop_kalman - app name changed to Genesys Bold Messenger

* develop_kalman - added app icons

* develop_kalman - change full demo app package name

* develop_kalman - removed prefilled accounts data

* develop_kalman - update application name

* develop_kalman - update the sample to work with the latest Bold version 4.8.3

### CODE REVIEW

File: SDKSamples/common/src/main/java/com/common/utils/chatForm/accounts.kt [hunk 1]
```diff
@@ -10,39 +10,39 @@ import java.util.UUID
 object Accounts {
```
Comment (tehilaR2) [inline SDKSamples/common/src/main/java/com/common/utils/chatForm/accounts.kt:10]: Accounts object purpose was to have default tests accounts for fast usage. If you think we should not provide default account we should think of removing it from usage. There is no reason to have it for creating empty accounts. (not for this PR )

File: SDKSamples/fulldemo/build.gradle [hunk 2]
```diff
@@ -55,11 +54,11 @@ dependencies {
...
     implementation project(':common')
 
-    /*implementation ("com.nanorep.devconversation:chatintegration:4.9.+"){changing = true}
-    implementation ("com.nanorep.devcore:accessibility:4.9.+"){changing = true}*/
+//    implementation ("com.nanorep.devconversation:chatintegration:4.8.+"){changing = true}
+//    implementation ("com.nanorep.devcore:accessibility:4.8.+"){changing = true}
 
-    implementation ("com.bold360ai-sdk.conversation:chatintegration:4.8.+"){changing = true}
-    implementation ("com.bold360ai-sdk.core:accessibility:4.8.+"){changing = true}
+    implementation ("com.bold360ai-sdk.conversation:chatintegration:4.8.3"){changing = true}
```
Comment (tehilaR2) [inline SDKSamples/fulldemo/build.gradle:60]: in case of using fixed version there is no reason to have the `{changing = true }`

**Review by tehilaR2 — CHANGES_REQUESTED** [review]

File: SDKSamples/common/src/main/java/com/common/utils/chatForm/accounts.kt (hunk 1 above)
Comment (KalmanPerkelGenesys) [reply SDKSamples/common/src/main/java/com/common/utils/chatForm/accounts.kt:10]: The reason for making them empty was to have a quick solution for not uploading the credentials to the store and removing placeholders. 

Should I create a Jira task to remove the Accounts class?

Comment (tehilaR2) [reply SDKSamples/common/src/main/java/com/common/utils/chatForm/accounts.kt:10]: let's have a ticket for that on the DX sample which has also the accounts object.
For the legacy, let's not put an effort on that.

**Review by tehilaR2 — APPROVED** [review]

File: SDKSamples/common/src/main/java/com/common/utils/chatForm/accounts.kt (hunk 1 above)
Comment (tehilaR2) [reply SDKSamples/common/src/main/java/com/common/utils/chatForm/accounts.kt:10]: Just remove the fame account since it's not in use.

**Pull request closed without merging**